### PR TITLE
Working on routing. In progres.

### PR DESCRIPTION
**Pull request by flat1ander**
Working on index route and other routes. Nothing to review at this time.

### CODE REVIEW

File: controllers/monsters.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const Monsters = require('../models/monsters')
+
```
Comment (sk1264) [inline controllers/monsters.js:4]: Comment testing.

File: controllers/monsters.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+// Index:
+router.get('/', async(req, res) => {
+    const monsters = await Monsters.find({});
```
Comment (dslvrgl8) [inline controllers/monsters.js:7]: comment testing

**Review by dslvrgl8 — APPROVED** [review]
Looks good

**Review by sk1264 — APPROVED** [review]
Looks FANTASTIC!

**Pull request merged**